### PR TITLE
Wire up attachment popup + download attachments when clicked

### PR DESCRIPTION
**Pull request by chromakode**
:eyeglasses: @keybase/react-hackers

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
The commits 73b98a1b7832...dd1569c9738d introduce visual changes on linux. <details><summary>:mag_right: 7 changed</summary>

 * Changed: **Popup_Dialog-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719453286-73b98a1b7832-dd1569c9738d-linux/Popup_Dialog-Normal-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719453286-73b98a1b7832-dd1569c9738d-linux/Popup_Dialog-Normal-dd1569c9738d.png)  
   Popup_Dialog-Normal comparison
 * Changed: **Attachment_Popup-You_Zoomed** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719453286-73b98a1b7832-dd1569c9738d-linux/Attachment_Popup-You_Zoomed-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719453286-73b98a1b7832-dd1569c9738d-linux/Attachment_Popup-You_Zoomed-dd1569c9738d.png)  
   Attachment_Popup-You_Zoomed comparison
 * Changed: **Attachment_Popup-You_Wide_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719453286-73b98a1b7832-dd1569c9738d-linux/Attachment_Popup-You_Wide_Image-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719453286-73b98a1b7832-dd1569c9738d-linux/Attachment_Popup-You_Wide_Image-dd1569c9738d.png)  
   Attachment_Popup-You_Wide_Image comparison
 * Changed: **Attachment_Popup-You_Small_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719453286-73b98a1b7832-dd1569c9738d-linux/Attachment_Popup-You_Small_Image-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719453286-73b98a1b7832-dd1569c9738d-linux/Attachment_Popup-You_Small_Image-dd1569c9738d.png)  
   Attachment_Popup-You_Small_Image comparison
 * Changed: **Attachment_Popup-You_Popup_Showing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719453286-73b98a1b7832-dd1569c9738d-linux/Attachment_Popup-You_Popup_Showing-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719453286-73b98a1b7832-dd1569c9738d-linux/Attachment_Popup-You_Popup_Showing-dd1569c9738d.png)  
   Attachment_Popup-You_Popup_Showing comparison
 * Changed: **Attachment_Popup-You** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719453286-73b98a1b7832-dd1569c9738d-linux/Attachment_Popup-You-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719453286-73b98a1b7832-dd1569c9738d-linux/Attachment_Popup-You-dd1569c9738d.png)  
   Attachment_Popup-You comparison
 * Changed: **Attachment_Popup-Following_Popup_Showing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719453286-73b98a1b7832-dd1569c9738d-linux/Attachment_Popup-Following_Popup_Showing-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719453286-73b98a1b7832-dd1569c9738d-linux/Attachment_Popup-Following_Popup_Showing-dd1569c9738d.png)  
   Attachment_Popup-Following_Popup_Showing comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 73b98a1b7832...7845bbead5e5 introduce visual changes on linux. <details><summary>:mag_right: 7 changed</summary>

 * Changed: **Popup_Dialog-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719586373-73b98a1b7832-7845bbead5e5-linux/Popup_Dialog-Normal-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719586373-73b98a1b7832-7845bbead5e5-linux/Popup_Dialog-Normal-7845bbead5e5.png)  
   Popup_Dialog-Normal comparison
 * Changed: **Attachment_Popup-You_Zoomed** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719586373-73b98a1b7832-7845bbead5e5-linux/Attachment_Popup-You_Zoomed-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719586373-73b98a1b7832-7845bbead5e5-linux/Attachment_Popup-You_Zoomed-7845bbead5e5.png)  
   Attachment_Popup-You_Zoomed comparison
 * Changed: **Attachment_Popup-You_Wide_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719586373-73b98a1b7832-7845bbead5e5-linux/Attachment_Popup-You_Wide_Image-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719586373-73b98a1b7832-7845bbead5e5-linux/Attachment_Popup-You_Wide_Image-7845bbead5e5.png)  
   Attachment_Popup-You_Wide_Image comparison
 * Changed: **Attachment_Popup-You_Small_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719586373-73b98a1b7832-7845bbead5e5-linux/Attachment_Popup-You_Small_Image-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719586373-73b98a1b7832-7845bbead5e5-linux/Attachment_Popup-You_Small_Image-7845bbead5e5.png)  
   Attachment_Popup-You_Small_Image comparison
 * Changed: **Attachment_Popup-You_Popup_Showing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719586373-73b98a1b7832-7845bbead5e5-linux/Attachment_Popup-You_Popup_Showing-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719586373-73b98a1b7832-7845bbead5e5-linux/Attachment_Popup-You_Popup_Showing-7845bbead5e5.png)  
   Attachment_Popup-You_Popup_Showing comparison
 * Changed: **Attachment_Popup-You** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719586373-73b98a1b7832-7845bbead5e5-linux/Attachment_Popup-You-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719586373-73b98a1b7832-7845bbead5e5-linux/Attachment_Popup-You-7845bbead5e5.png)  
   Attachment_Popup-You comparison
 * Changed: **Attachment_Popup-Following_Popup_Showing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719586373-73b98a1b7832-7845bbead5e5-linux/Attachment_Popup-Following_Popup_Showing-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484719586373-73b98a1b7832-7845bbead5e5-linux/Attachment_Popup-Following_Popup_Showing-7845bbead5e5.png)  
   Attachment_Popup-Following_Popup_Showing comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 73b98a1b7832...f88045dddd97 introduce visual changes on linux. <details><summary>:mag_right: 7 changed</summary>

 * Changed: **Popup_Dialog-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484721117169-73b98a1b7832-f88045dddd97-linux/Popup_Dialog-Normal-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484721117169-73b98a1b7832-f88045dddd97-linux/Popup_Dialog-Normal-f88045dddd97.png)  
   Popup_Dialog-Normal comparison
 * Changed: **Attachment_Popup-You_Zoomed** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484721117169-73b98a1b7832-f88045dddd97-linux/Attachment_Popup-You_Zoomed-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484721117169-73b98a1b7832-f88045dddd97-linux/Attachment_Popup-You_Zoomed-f88045dddd97.png)  
   Attachment_Popup-You_Zoomed comparison
 * Changed: **Attachment_Popup-You_Wide_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484721117169-73b98a1b7832-f88045dddd97-linux/Attachment_Popup-You_Wide_Image-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484721117169-73b98a1b7832-f88045dddd97-linux/Attachment_Popup-You_Wide_Image-f88045dddd97.png)  
   Attachment_Popup-You_Wide_Image comparison
 * Changed: **Attachment_Popup-You_Small_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484721117169-73b98a1b7832-f88045dddd97-linux/Attachment_Popup-You_Small_Image-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484721117169-73b98a1b7832-f88045dddd97-linux/Attachment_Popup-You_Small_Image-f88045dddd97.png)  
   Attachment_Popup-You_Small_Image comparison
 * Changed: **Attachment_Popup-You_Popup_Showing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484721117169-73b98a1b7832-f88045dddd97-linux/Attachment_Popup-You_Popup_Showing-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484721117169-73b98a1b7832-f88045dddd97-linux/Attachment_Popup-You_Popup_Showing-f88045dddd97.png)  
   Attachment_Popup-You_Popup_Showing comparison
 * Changed: **Attachment_Popup-You** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484721117169-73b98a1b7832-f88045dddd97-linux/Attachment_Popup-You-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484721117169-73b98a1b7832-f88045dddd97-linux/Attachment_Popup-You-f88045dddd97.png)  
   Attachment_Popup-You comparison
 * Changed: **Attachment_Popup-Following_Popup_Showing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484721117169-73b98a1b7832-f88045dddd97-linux/Attachment_Popup-Following_Popup_Showing-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484721117169-73b98a1b7832-f88045dddd97-linux/Attachment_Popup-Following_Popup_Showing-f88045dddd97.png)  
   Attachment_Popup-Following_Popup_Showing comparison
</summary>

File: shared/actions/chat.js [hunk 1]
```diff
@@ -166,6 +167,20 @@ function selectConversation (conversationIDKey: ConversationIDKey, fromUser: boo
   return {type: 'chat:selectConversation', payload: {conversationIDKey, fromUser}}
 }
 
+function openAttachmentPopup (message: AttachmentMessage): AsyncAction {
```
Comment (chrisnojima) [inline shared/actions/chat.js]: why aren't we using a saga?

Comment (chromakode) [inline shared/actions/chat.js]: @chrisnojima fixed!

**Comment by keybase-ci-visdiff** [comment]
The commits 73b98a1b7832...fd6416a6c94f introduce visual changes on linux. <details><summary>:mag_right: 7 changed</summary>

 * Changed: **Popup_Dialog-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484766298969-73b98a1b7832-fd6416a6c94f-linux/Popup_Dialog-Normal-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484766298969-73b98a1b7832-fd6416a6c94f-linux/Popup_Dialog-Normal-fd6416a6c94f.png)  
   Popup_Dialog-Normal comparison
 * Changed: **Attachment_Popup-You_Zoomed** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484766298969-73b98a1b7832-fd6416a6c94f-linux/Attachment_Popup-You_Zoomed-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484766298969-73b98a1b7832-fd6416a6c94f-linux/Attachment_Popup-You_Zoomed-fd6416a6c94f.png)  
   Attachment_Popup-You_Zoomed comparison
 * Changed: **Attachment_Popup-You_Wide_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484766298969-73b98a1b7832-fd6416a6c94f-linux/Attachment_Popup-You_Wide_Image-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484766298969-73b98a1b7832-fd6416a6c94f-linux/Attachment_Popup-You_Wide_Image-fd6416a6c94f.png)  
   Attachment_Popup-You_Wide_Image comparison
 * Changed: **Attachment_Popup-You_Small_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484766298969-73b98a1b7832-fd6416a6c94f-linux/Attachment_Popup-You_Small_Image-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484766298969-73b98a1b7832-fd6416a6c94f-linux/Attachment_Popup-You_Small_Image-fd6416a6c94f.png)  
   Attachment_Popup-You_Small_Image comparison
 * Changed: **Attachment_Popup-You_Popup_Showing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484766298969-73b98a1b7832-fd6416a6c94f-linux/Attachment_Popup-You_Popup_Showing-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484766298969-73b98a1b7832-fd6416a6c94f-linux/Attachment_Popup-You_Popup_Showing-fd6416a6c94f.png)  
   Attachment_Popup-You_Popup_Showing comparison
 * Changed: **Attachment_Popup-You** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484766298969-73b98a1b7832-fd6416a6c94f-linux/Attachment_Popup-You-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484766298969-73b98a1b7832-fd6416a6c94f-linux/Attachment_Popup-You-fd6416a6c94f.png)  
   Attachment_Popup-You comparison
 * Changed: **Attachment_Popup-Following_Popup_Showing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484766298969-73b98a1b7832-fd6416a6c94f-linux/Attachment_Popup-Following_Popup_Showing-73b98a1b7832.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484766298969-73b98a1b7832-fd6416a6c94f-linux/Attachment_Popup-Following_Popup_Showing-fd6416a6c94f.png)  
   Attachment_Popup-Following_Popup_Showing comparison
</summary>

**Pull request merged**